src/Arcus.Templates.AzureFunctions.Http/.template.config/template.json

```
@@ -0,0 +1,73 @@
+﻿{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Arcus",
+  "classifications": [
+    "AzureFunctions",
+    "Http",
+    "API"
+  ],
+  "name": "Arcus Azure Functions Http v#{Package.Version}#",
+  "identity": "Arcus.Templates.AzureFunctions.Http",
+  "shortName": "arcus-az-http",
```

Let's make it more clear what we are creating in this template:
```suggestion
  "shortName": "arcus-az-func-http",
```